src/shared/SoundPlayButton.js

```diff
@@ -31,13 +32,26 @@ const SoundPlayButton = ({ soundSource }) => {
   };
 
   return (
-    <Icon
-      name="play-circle-o"
-      style={isPressed ? { color: "#7d8b91" } : { color: "#EBEBEB" }}
-      size={150}
-      onPress={() => playSound()}
-    />
+    <ButtonContainer onPress={() => playSound()}>
+      <Icon
+        name="volume-up"
+        style={isPressed ? { color: "#7d8b91" } : { color: "#fff" }}
```

Comment:
do you still need this now that it is inside the `TouchableHighlight`? Can you try removing it and see if it still changes colors when you press it?